model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,140 @@
+          variable `VCAP_APPLICATION.space_id`. This is the same value as
+          reported by `cf space <space-name> --guid`.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: org.name
+        type: string
+        brief: >
+          The name of the CloudFoundry organization the app is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.org_name`. This is the same value as
+          reported by `cf orgs`.
+        examples: ['my-org-name']
+      - id: org.id
+        type: string
+        brief: >
+          The guid of the CloudFoundry org the application is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.org_id`. This is the same value as
+          reported by `cf org <org-name> --guid`.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: source.type
+        type: string
+        brief: >
+          The type of the event source.
+        note: |
+          CloudFoundry generates log messages with different source types for an
+          applications. This indicates, whether the log was created from the
+          actual application, a background task, the central load balancer, or
+          the CloudFoundry runtime, e.g. during a deployment.
+
+          The `source_type` is contained as a tag of the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is also available in the syslog drain structured data as documented
+          in the [CloudFoundry Dev Guide](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html).
+          Application developers can access the value of the `source_type` in
+          the log stream generated by `cf logs <app-name>`, usually in square
+          brackets.
+        examples:
+          - 'APP/PROC/WEB/0'
+          - 'APP/TASK/jobscheduler-task-70013817-ab92-4f75-82d4-de8ae4dd64b4'
+          - 'RTR'
+          - 'STG'
+      - id: process.id
+        type: string
+        brief: >
+          The UID identifying the process.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.process_id`. It is supposed to be equal to
+          `VCAP_APPLICATION.app_id` for applications deployed to the runtime.
+          For system components, this could be the actual PID.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: process.type
+        type: string
+        brief: >
+          The type of process.
+        note: |
+          CloudFoundry applications can consist of multiple jobs. Usually the
+          main process will be of type `web`. There can be additional background
+          tasks or side-cars with different process types.
+        examples: ['web']
```

Comment:
I wonder if `process.id` and `process.type` could be general process attributes https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/process.md#process

Reply:
The CloudFoundry process types have very particular values. The do not indicate the executable nor runtime. I do not find a matching field in the process resource attributes.

Reply:
Hi @joaopgrassi, thank you very much for the tips and the review. Tbh, I've always hated the different meanings and different values of some tags depending on the context be it telemetry data type: gauge, counter, log etc or the source CF apps or platform components. It's a technical debt that we have in Cloud Foundry. We cannot fix it for our current consumers and current formats, but we can clean up the things the consumption with OpenTelemetry :)

IMHO, we should define everything properly according to the other existing rules and semantic conventions in OpenTelemetry, so that we remove the dual meaning and the confusion with some tags and add a custom reader or processor in the Otel Collector which we deliver with CF to read the original CF formatted data and reformat everything properly according to the semantic conventions which we will define with this PR. 

I suggest that we (me, @ctlong and the other maintainers in the Logging and Metrics Team in Cloud Foundry) sit together, take a look what we have and decide how to handle the different parameters. We will than come back with all the meanings of different values for some parameters and possibly propose proper Otel attributes or we'll ask for help.

@ctlong I hope you agree with my suggestion, I've already started collecting and analyzing telemetry data from the platform components and some apps. I'll contact you when I have gathered all the needed data.

Reply:
@chombium , @ctlong: If you discuss, will you provide a suggestion about `source.id` and `instance.id` as well? After all, these two attributes also carry a dual meaning.

Reply:
Since @chombium approved this PR, I resolve this issue for now.